packages/ledger_design_system/lib/ledger_theme/widgets/svg_themed_icon.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:ledger_design_system/ledger_design_system.dart';
+
+class SvgThemedIcon extends StatelessWidget {
+  const SvgThemedIcon({
+    required this.svgPath,
+    this.size,
+    this.color,
+    this.semanticsLabel,
+    this.package,
```

Comment:
@alphapokedex do we need this `package` exposed to public API since this is part of only ledger design system.